### ISSUE TITLE
Re-write ./scripts/gpg-init Bash script in Python

### ISSUE THREAD

**Issue by romanz** [romanz/trezor-agent#148]
This way it'll be available after running `pip install trezor_agent`.

**Comment by galuszkak** [comment]
@romanz If I can add one more recommendation here, now it takes whatever is in PATH as python. Could it be parametrized? I'm mostly using python 3 so I had to just change that line there.
```
GPG_BINARY=$(python -c "import libagent.gpg.keyring as k; print(k.get_gnupg_binary())")
```

**Comment by galuszkak** [comment]
Ah, disregard my comment, this ticket about rewriting that script to Python. Didn't catch that, I'm probably to sick, should stay away from computer ;).

**Comment by romanz** [comment]
No problem - get well soon :)

**Comment by romanz** [comment]
Need to update the documentation and bump the version as well.

**Comment by romanz** [comment]
Should be fixed by #150 PR.
